### PR TITLE
 move "Warn about duplicates on import" preferences option

### PR DESCRIPTION
**Pull request by christinait2022**
fixes: [koppor#570](https://github.com/koppor/jabref/issues/570)
move "Warn about duplicates on import" option from  ```Perferences -> File``` to ```Perferences -> Import and Export```.

图片

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by christinait2022** [comment]
图片

**Comment by Siedlerchr** [comment]
Please add a changelog entry as well

**Review by Siedlerchr — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +50,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where the exception that there are invalid characters in filename. [#8786](https://github.com/JabRef/jabref/issues/8786)
 - When the proxy configuration removed the proxy user/password, this change is applied immediately.
 - We fixed an issue where removing several groups deletes only one of them. [#8390](https://github.com/JabRef/jabref/issues/8390)
+- We fixed an issue where "Warn about duplicates on import" option now under Perferences -> Import and Export. [kopper#570](https://github.com/koppor/jabref/issues/570)
```
Comment (Siedlerchr) [inline CHANGELOG.md:53]:
```suggestion
- We fixed an issue where "Warn about duplicates on import" option now under Perferences -> Import and Export. [kopper#570](https://github.com/koppor/jabref/issues/570)
```

**Comment by Siedlerchr** [comment]
Thanks for your contribution, I just adapted the changelog a bit.

**Review by koppor — APPROVED** [review]

**Pull request merged**